back/engines/free/email_campaigns/app/views/email_campaigns/moderator_digest_mailer/campaign_mail.mjml

```diff
@@ -94,6 +94,6 @@
     </mj-column>
   </mj-section>
 
-  <%= render partial: 'application/cta_button', locals: { href:  "https://#{event.tenant_host}/admin/projects", message: format_message('cta_moderate') } %>
+  <%= render partial: 'application/cta_button', locals: { href:  "https://#{AppConfiguration.instance.host}/admin/projects", message: format_message('cta_moderate') } %>
```

Comment:
See PR description for explanation & discussion of this change.

Reply:
> Not sure, but it might be possible to get the app configuration through `@app_configuration`

Maybe, but using `url_service.home_url(app_configuration: @app_configuration, locale: @locale)` gives an `http://...` result. Not sure if it will do this for active platform, or only in local dev. Looking into it.

Reply:
I didn't find a way to test it via console on an active platform, but I think I'll go with your idea on the assumption it will return `https` in use on an active platform, since as you say it is already in use in the user digest mailer.

We can always check after it's released, since this isn't going to break anything else.